autogen.sh

```diff
@@ -1,5 +1,19 @@
-#! /bin/sh
+#! /bin/bash
 
 test -d ./config || mkdir ./config
 
+FILES=./man/*.1
+printf "dist_man1_MANS = " > ./man/Makefile.am
```

Comment:
It would be better to use ```echo -n```, which is a bash builtin, rather than ```printf```, which is a separate executable that might not be present on all systems.